src/components/pages/HomePage/HowPulsarWorks/HowPulsarWorks.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import s from './HowPulsarWorks.module.css'
+import ScreenTitle from '../ui/ScreenTitle/ScreenTitle';
+
+// SVGO breaks the illustration. To fix it, we import it as is.
+import illustrationDesktop from '!!raw-loader!./img/illustration-desktop.svg';
+import illustrationMobile from '!!raw-loader!./img/illustration-mobile.svg';
+
+import BookkeeperIcon from './img/bookkeeper.svg';
+import BrokersIcon from './img/brokers.svg';
+import ZookeeperIcon from './img/zookeeper.svg';
+import ProducerAndConsumerIcon from './img/producer-and-consumer.svg';
+import Slider from '@site/src/components/ui/Slider/Slider';
+
+const cards: CardProps[] = [
+  {
+    title: 'Producer & Consumer',
+    image: <ProducerAndConsumerIcon />,
+    children: (
+      <p>
+        A Pulsar client contains a consumer and a producer.
+        A producer writes messages on a topic.
+        A consumer reads messages from a topic and acknowledges specific messages or all up to a specific message.
+      </p>
+    )
+  },
+  {
+    title: 'Apache Zookeeper',
+    image: <ZookeeperIcon />,
+    children: (
+      <p>
+        Pulsar and BookKeeper use Apache ZooKeeper to save metadata coordinated between nodes,
+        such as a list of ledgers per topic, segments per ledger, and mapping of topic bundles to a broker.
+        It’s a cluster of highly available and replicated servers (usually 3).
```

Comment:
```suggestion
        It's a cluster of highly available and replicated servers (usually 3).
```

I remember we keep using `'` consistently but it can be a tribal knowledge without publicly acknowledge. cc @Anonymitaet what do you think?

Anyway, it won't block this PR from merging.

Reply:
Good catch!

`’` and `'` intend to indicate possessions, but they look slightly different due to the user's typing methods.

Both are OK. If we need to keep it consistent, I suggest using `'` because it appears more often in current `.md` docs. It's easier for us to fix the rest (if needed).

<img width="559" alt="image" src="https://github.com/apache/pulsar-site/assets/50226895/0dc5206e-73d2-4c83-b942-b1d4899de39d">

<img width="507" alt="image" src="https://github.com/apache/pulsar-site/assets/50226895/9e98947a-6523-4ffb-927e-9c1b47ad3a8a">

Reply:
I just tried - I have no idea what key on my keyboard gives me ’ :)